package/tests/test_server.py

```diff
@@ -282,7 +296,6 @@ def test_no_load_file(cli_runner):
     """
     result = cli_runner.invoke(server.commands, ["viz"])
     assert result.exit_code == 1
-    assert "Could not find a Kedro project root." in result.output
```

Comment:
Can you explain a bit why this is needed?

Reply:
Yes, previously this will fail here but now it fails earlier. It will try to load the `settings` from a non-existing module and will throw ImportError.